### ISSUE TITLE
Windows: after_files list is not generated

### ISSUE THREAD

**Issue by reiver-dev** [wbthomason/packer.nvim#336]
- `nvim --version`:
```
NVIM v0.5.0-dev+1291-gb227cedf8
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: C:/Program Files (x86)/Microsoft Visual Studio/2017/Enterprise/VC/Tools/MSVC/14.16.27023/bin/Hostx86/x64/cl.exe /DWIN32 /D_WINDOWS /W3 /MD /Zi /O2 /Ob1 /DNDEBUG /W3 -D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_DEPRECATE -DWIN32 -D_WIN32_WINNT=0x0600 -DINCLUDE_GENERATED_DECLARATIONS -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -ID:/a/neovim/neovim/build/config -ID:/a/neovim/neovim/src -ID:/a/neovim/neovim/nvim-deps/usr/include -ID:/a/neovim/neovim/build/src/nvim/auto -ID:/a/neovim/neovim/build/include
Compiled by runneradmin@fv-az152-644

Features: -acl +iconv +tui
```

- `git --version`:
```
bc6feab6b3c1ad5af9aed077857aed8c4e4657f1
```

- Operating system/version:
```
Window 10
Major  Minor  Build  Revision
-----  -----  -----  --------
10     0      19042  0
```

- Terminal name/version:
```
Windows Terminal Preview
Version: 1.8.1032.0
```

### Steps to reproduce
Install nvim-compe to opt. Generate packer_compiled.vim
```
nvim --headless -u NORC --noplugin +"packadd packer.nvim"  +"lua require([[packer]]).startup(function(use) use { [[hrsh7th/nvim-compe]], as = [[compe]], opt = true }; end)" +PackerInstall +PackerCompile  +q
```

### Actual behaviour
after_files not present in packer_compiled.vim

### Expected behaviour
after_files successfully generated and loaded

### Suspected reason
On Windows glob function ignores first directory (plugin) when backslash is used
```
$> nvim --headless -u NORC --noplugin +"echo glob(stdpath('data') . '\site\pack\packer\opt\compe\after\plugin\**\*.vim', v:false, v:true)" +q 
[]
```
```
$> nvim --headless -u NORC --noplugin +"echo glob(stdpath('data') . '\site\pack\packer\opt\compe\after\plugin\**/*.vim', v:false, v:true)" +q 
[ file found ]
```
Next issue will arise at load.lua
```
      if plugin.after_files then
        for _, file in ipairs(plugin.after_files) do
          cmd('silent exe "source ' .. file .. '"')
        end
      end
```
exe will have different escaping rules to shell commands and windows backslashes will be treated as start of escape sequences. Chaning ```silent exe "source ..."``` to ```silent source fnameescape(file)``` seem to fix the issue.

**Comment by wbthomason** [comment]
@reiver-dev: Sorry for the lag in dealing with this issue - it somehow slipped through my inbox. As I commented on #384, I'm a bit surprised that the changes regarding forward/backward slashes in patterns work - I assume that you've tested these changes on Windows?

**Comment by reiver-dev** [comment]
As commented on the PR #384 

I have checked slash part also with vim8.2. Vim demonstrates same behavior.

I suppose it is caused by:

https://github.com/neovim/neovim/blob/30befcdce47b4964b4d2a5aeb078ddec8e2fcc0b/src/nvim/path.c#L665
https://github.com/vim/vim/blob/3d9c4eefe656ee8bf58c0496a48bd56bac180056/src/filepath.c#L3595

**Comment by wbthomason** [comment]
Ok, thanks! I'm OK merging #384, though I'd like to revert the forward slash hardcoding if the upstream bug ever gets fixed. If you're willing, it would be great if you could file an upstream issue for Neovim/Vim!

**Comment by wbthomason** [comment]
Closed via #384.
